purchase_last_price_info/models/product_product.py

```diff
@@ -0,0 +1,84 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# Copyright 2019 ForgeFlow S.L.
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    last_purchase_line_ids = fields.One2many(
+        comodel_name="purchase.order.line",
+        inverse_name="product_id",
+        domain=[("state", "in", ["purchase", "done"])],
+        string="Last Purchase Order Lines",
+    )
+    last_purchase_line_id = fields.Many2one(
+        comodel_name="purchase.order.line",
+        compute="_compute_last_purchase_line_id",
+        string="Last Purchase Line",
+        store=True,
+    )
```

Comment:
Also, can you explain a bit better why these 2 fields are necessary and how they are set up? I don't see them in the view, so maybe a quick explanation in the readme would help. OTOH, why does `last_purchase_line_id` need to be stored? If you are just getting the last one, maybe a simple compute is enough, right? Or does it have a big influence in performance?

Reply:
Now `last_purchase_line_ids` field is necessary to auto-set `last_purchase_line_id` field (when `last_purchase_line_ids` have changed according to new purchase lines), and these is the explanation to set as compute and `store=True`.

Reply:
Ok, I didn't think about the `inverse_name="product_id"` and I didn't know where `last_purchase_line_ids` came from :sweat_smile: 
However, I still don't understand the need for the `store=True` in `last_purchase_line_id`. Can you explain a little better?